website/docs/d/vgpu_profile.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vgpu_profile"
+sidebar_current: "docs-vcd-datasource-vgpu-policy"
+description: |-
+  Provides a datasource to read vGPU profiles in VMware Cloud Director.
+---
+
+# vcd\_vm\_vgpu\_profile
+
+Supported in provider *3.11* and VCD *10.4.0+*.
```

Comment:
```suggestion
Experimental in provider *3.11*.
```
VCD version is not needed, since 3.11 supports 10.4.0+

Reply:
Fixed